epoch_multinode_test.go

```diff
@@ -271,6 +295,23 @@ func (n *inMemNetwork) addNode(node *testNode) {
 	n.instances = append(n.instances, node)
 }
 
+func (n *inMemNetwork) IsConnected(node NodeID) bool {
```

Comment:
It will be faster if we look at if the node is *not* connected. This way we don't need to iterate over the instances each time and they can be an empty list by default.